rust/helpers.c

```diff
@@ -80,7 +80,3 @@ void rust_helper_kunmap(struct page *page)
 	return kunmap(page);
 }
 EXPORT_SYMBOL_GPL(rust_helper_kunmap);
-
-// See https://github.com/rust-lang/rust-bindgen/issues/1671
-static_assert(__builtin_types_compatible_p(size_t, uintptr_t),
-	"size_t must match uintptr_t, what architecture is this??");
```

Comment:
I saw your other PR -- I guess this is fine, but perhaps it doesn't hurt to leave it just in case...

Reply:
On 32-bit arm, size_t doesn't resolve to the same underlying type as uintptr_t, so the static assert will be triggered. Maybe I should make that clearer in the commit msg.

Reply:
Under `#[cfg]` perhaps?